libs/sx1302_hal/patches/000-edit_cflags.patch

```diff
@@ -0,0 +1,42 @@
+From 389e8138f543baf334442ca52edf5f7b4cfe13dc Mon Sep 17 00:00:00 2001
+From: Marcus Schref <mschref@web.de>
+Date: Mon, 22 Aug 2022 10:02:13 +0200
+Subject: [PATCH] sx1302_hal: edit cflags
+
+Enable use of TARGET_CFLAGS in addition to specified CFLAGS
```

Comment:
Did you try to upstream this? Otherwise, we will need to carry this patch forever.

Reply:
I opened pull requests for both sx1302_hal patches.
https://github.com/Lora-net/sx1302_hal/pull/85
https://github.com/Lora-net/sx1302_hal/pull/86